src/hb-directwrite.hh

```diff
@@ -237,7 +237,7 @@ HB_INTERNAL hb_directwrite_global_t *
 get_directwrite_global ();
 
 HB_INTERNAL IDWriteFontFace *
-dw_face_create (hb_blob_t *blob, unsigned index);
+dw_face_create (hb_blob_t *blob, unsigned index, hb_font_t *font);
```

Comment:
This is a confusing function signature. blob&index typically mean a face. Can't you get that out from the font already? Why Both?

Reply:
It is called in two places, one has a font, the other only has a blob and index.

Reply:
Sounds like two different functions to me :)). What's up?

I'm still digesting, but I think an IDWriteFontFace maps to hb-font-t indeed.

Reply:
Yes. I’m starting to think it was a mistake to make `hb_directwrite_font_create()` take an `IDWriteFont`, it should have been `IDWriteFontFace`.